handbook/engineering/onboarding/index.md

```diff
@@ -0,0 +1,41 @@
+# Engineering onboarding
+
+Welcome! We're excited to have you join the team. This document outlines the structure of your first few weeks at Sourcegraph.
+
+## Getting set up
+
+You'll have to get some basics set up in your first few days:
+
+- Complete [general onboarding](../../people-ops/onboarding/index.md#general-onboarding-checklist)
+- [Configure your GitHub notifications.](../github-notifications/index.md)
+- Join #dev-announce, #dev-chat, and your team's channel on Slack, as well as any other channels you find interesting. [Team chat documentation](../../communication/team_chat.md#engineering)
+- Set up your [local development environment](https://github.com/sourcegraph/sourcegraph/blob/main/doc/dev/getting-started/index.md). If you encounter any issues, ask for help in Slack and then update the documentation to reflect the resolution (so the next engineer that we hire doesn't run into the same problem).
+- [Add Sourcegraph as a browser search engine](https://docs.sourcegraph.com/integration/browser_search_engine). Add another entry for our internal instance: `https://sourcegraph.sgdev.org/search?q=%s`. This also ensures you have access to our internal instance.
+- Read through your team's handbook page, to learn about the team and its internal processes.
+- Complete the onboarding for your role:
```

Comment:
I like the cross-reference but it also feels a bit recursive as `index.md -> *-onboarding.md -> index.md#getting-set-up -> *-onbarding.md`.

Reply:
Yeah, there is a loop here, but I think it is valuable to link back to more general onboarding as a first step in case someone lands on the leaf page.